app/components/facility/addFacilitySchema.ts

```diff
@@ -0,0 +1,70 @@
+import {JSONSchema6} from 'json-schema';
+
+const addFacilitySchema: JSONSchema6 = {
+  type: 'object',
+  title: 'Facility Details',
+  properties: {
```

Comment:
I would only keep the facility name and BC GHG ID. The other information doesn't really need to be in the facility table